sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveParquetMetastoreSuite.scala

```diff
@@ -512,16 +512,14 @@ class HiveParquetMetastoreSuite extends ParquetPartitioningTest {
         |PARTITION (`date`='2015-04-01')
         |select a, b from jt
       """.stripMargin)
-    // Right now, insert into a partitioned Parquet is not supported in data source Parquet.
-    // So, we expect it is not cached.
-    assert(getCachedDataSourceTable(tableIdentifier) === null)
+    checkCached(tableIdentifier)
```

Comment:
This change also affects ORC. Could you add test for Orc?

Reply:
I will check existing tests. ORC cases are not that much probably, I will add similar cases for ORC then.